.gitignore

```diff
@@ -1,3 +1,5 @@
 /taskcluster-base-test.conf.json
 node_modules/
 .vagrant/
+.test/
+lib/
```

Comment:
@jhford didn't want to do this, since npm copies .gitignore in the absence of an .npmignore, and he didn't want to add an .npmignore